File-Formats/Presentation/Working-with-sections.md

```diff
@@ -656,145 +480,102 @@ The following code example demonstrates how to clone the slide collection of a s
 foreach (ISlide slide in slides)
-
-pptxDoc.Slides.Add(slide);
-
+    pptxDoc.Slides.Add(slide);
 //Saves the PowerPoint presentation
 pptxDoc.Save("Sections.PPTX");
-
 {% endhighlight %}
 
 {% highlight vb.net tabtitle="VB.NET" %}
-
 'Loads a PowerPoint presentation
 Dim pptxDoc As IPresentation = Presentation.Open("PPTXWithSections.PPTX")
-
 'Clones the slides in 3rd section
 Dim slides As ISlides = pptxDoc.Sections(2).Clone()
-
 'Creates a destination PowerPoint presentation instance. Existing presentations can also be used here.
 pptxDoc = Presentation.Create()
-
 'Iterates the cloned slides and adds the slides to the destination presentation
-For Each slide As ISlide In slides
-
-pptxDoc.Slides.Add(slide)
-
+    For Each slide As ISlide In slides
+    pptxDoc.Slides.Add(slide)
 Next
-
 'Save the PowerPoint presentation
 pptxDoc.Save("Sections.PPTX")
-
 {% endhighlight %}
 
 {% highlight c# tabtitle="UWP" %}
-
 //Instantiates the File Picker
 FileOpenPicker openPicker = new FileOpenPicker();
 openPicker.SuggestedStartLocation = PickerLocationId.Desktop;
 openPicker.FileTypeFilter.Add(".pptx");
-
 //Creates a storage file from FileOpenPicker
 StorageFile inputStorageFile = await openPicker.PickSingleFileAsync();
-
 //Loads or open an PowerPoint Presentation
 IPresentation pptxDoc= await Presentation.OpenAsync(inputStorageFile);
-
 //Clones the slides in 3rd section
 ISlides slides = pptxDoc.Sections[2].Clone();
-
 //Creates a destination PowerPoint presentation instance. Existing presentations can also be used here.
 pptxDoc = Presentation.Create();
-
 //Iterates the cloned slides and adds the slides to the destination presentation
 foreach (ISlide slide in slides)
-   pptxDoc.Slides.Add(slide);
-
+    pptxDoc.Slides.Add(slide);
 //Initializes FileSavePicker
 FileSavePicker savePicker = new FileSavePicker();
 savePicker.SuggestedStartLocation = PickerLocationId.Desktop;
 savePicker.SuggestedFileName = "Sections";
 savePicker.FileTypeChoices.Add("PowerPoint Files", new List<string>() { ".pptx" });
-
 //Creates a storage file from FileSavePicker
 StorageFile storageFile = await savePicker.PickSaveFileAsync();
-
 //Saves changes to the specified storage file
 await pptxDoc.SaveAsync(storageFile);
-
 {% endhighlight %}
 
 {% highlight c# tabtitle="ASP.NET Core" %}
-
 //Loads or open an PowerPoint Presentation
 FileStream inputStream = new FileStream("PPTXWithSections.PPTX",FileMode.Open);
 IPresentation pptxDoc = Presentation.Open(inputStream);
-
 //Clones the slides in 3rd section
 ISlides slides = pptxDoc.Sections[2].Clone();
-
 //Creates a destination PowerPoint presentation instance. Existing presentations can also be used here.
 pptxDoc = Presentation.Create();
-
 //Iterates the cloned slides and adds the slides to the destination presentation
 foreach (ISlide slide in slides)
-
-pptxDoc.Slides.Add(slide);
-
-//Save the PowerPoint Presentation as stream
+    pptxDoc.Slides.Add(slide);
+    //Save the PowerPoint Presentation as stream
```

Comment:
Please remove empty space before this line.

Reply:
@Suriya-Balamurugan -Done